scripts/release-tools/clusters.sh

```diff
@@ -212,6 +229,62 @@ create_qa_gke_cluster() {
   kubectl -n stackrox get pods
 }
 
+create_long_running_cluster() {
+
+  CLUSTER_NAME="$(get_cluster_name long-running)"
+  export CLUSTER_NAME
+
+  echo "cluster_name= $CLUSTER_NAME"
+  
+  status="$(check_cluster_status "$CLUSTER_NAME")"
+  
+  if does_cluster_exist "$CLUSTER_NAME"; then
+      echo "Unable to create cluster"
+  else
+      infractl create gke-default $CLUSTER_NAME --lifespan 168h --arg nodes=5 --wait --slack-me
+  fi
+  
+  # Set your local kubectl context to the remote cluster once the above completes successfully.
+  infractl get $CLUSTER_NAME --json | jq '.Connect' -r | bash
+
+  kubectl -n stackrox get pod
+  
+  
+  export MAIN_IMAGE_TAG=$(git describe --tags --abbrev=0) # Release version, e.g. 3.63.0-rc.2.
```

Comment:
Can't we use the variables `RELEASE PATCH_NUMBER RC_NUMBER` here?

Reply:
Now I am sure this will not work, because  I have tried it :)

```
    Image:      quay.io/stackrox-io/main:3.70.x
```

Reply:
I now use the environment variables.